target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-qxwlan-e2600ac-c1.dts

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2015, The Linux Foundation. All rights reserved.
```

Comment:
From https://openwrt.org/submitting-patches#dts_checklist

> Don't forget to add proper license, consider adding SPDX-License-Identifier: GPL-2.0-or-later OR MIT

I'm not a lawyer, but I think, that you can freely replace those texts with [appropriate simple line](https://github.com/openwrt/openwrt/commit/221d3e023451cad0b60b8a47e8b788d6368108c2#diff-7ecb7639c5199e391573618b0e444894). BTW you still retain copyright for your work as it's visible from commit message.